webapp/config/packages/nelmio_api_doc.yaml

```diff
@@ -1,9 +1,43 @@
 nelmio_api_doc:
     documentation:
+        tags:
```

Comment:
Would be better/cool if we can write some event listener or compiler pass to add this based on the actual tags in all files so we don't repeat ourself.

Reply:
Yes, It might be nice if this just comes with NelmioDoc. I've checked and this is actually only optional so I think we can take this out for now.